tests/testthat/test-base.R

```diff
@@ -338,3 +338,16 @@ test_that("Base S3 classes can be properties", {
   expect_error(Foo(x = 1), "@x must be S3<POSIXt>, not <double>")
 
 })
+
+
+test_that("inherits() works with S7_base_class", {
+  # nameOfClass() introduced in R 4.3
+  skip_if(getRversion() < "4.3")
+
+  # test nameOfClass.S7_base_class
+  expect_true(inherits("foo", class_character))
+
+  Foo := new_class(class_character)
```

Comment:
`:=` ??? 🤯

Reply:
Credit goes to @lawremi for the idea!

Reply:
When can we graduate it from a test helper?  :wink: It's super convenient and clarifies code.

Reply:
I'm on board for exporting it. (From the tidyverse, we'd have to figure out how to resolve the clash with  `` rlang`:=` ``)

Reply:
Yea, I think as discussed, the rlang one expects different types of arguments (symbols vs. calls) so disambiguating the intent should be possible.